client/directives/file-filter.js

```diff
@@ -0,0 +1,40 @@
+const isFileInput = el => el.tagName === 'INPUT' || el.type === 'file';
+const isProduction = process.env.NODE_ENV === 'production';
+
+export const install = Vue => {
+  const warn = (msg, vm) => Vue.util.warn(`[v-filefilter]: ${msg}`, vm);
```

Comment:
Since you're always using warn in conjunction with `isProduction` check, you could've make check a part of `warn` method.

Reply:
I might but this looks more _standardish_ from directive authoring perspective 🙃